### ISSUE TITLE
Disable numeral prefixing for some commands

### ISSUE THREAD

**Issue by hejops** [tridactyl/tridactyl#2308]
Tridactyl version: 1.17.1pre3763-e61fd83
Firefox version: Mozilla Firefox 75.0
Operating system: linux

*   Brief description of the problem:
I use the number keys to skip ahead in YouTube videos, but when want to I close the tab, "1d/2d/3d/..." is picked up, which closes the 1st/2nd/3rd/... tab. I've unbound 1d/2d/3d/..., but this behaviour persists; is there a way to prevent certain commands (in this case tabclose) from being "prefixed" with a number?

*   Config:
https://gist.githubusercontent.com/hejops/5e87d3381afc818b48b8c27d94790f8f/raw/fee0e5e9eeeba37ea6ba571e192ce029a9eb2c18/gistfile1.txt

**Comment by bovine3dom** [comment]
Does `bind d jsb tri.excmds.tabclose()` help?

**Comment by hejops** [comment]
It does prevent the numbered tab from closing at least, though it also produces `controller in excmd: Error: unexpected token: numeric literal`, which means an extra keypress to actually close the tab. Perhaps there's a way to suppress this error - preferably only for this specific command?

**Comment by bovine3dom** [comment]
I think `bind d jsb -p (JS_ARG) => tri.excmds.tabclose()` should do the trick.

**Comment by hejops** [comment]
Hmm, that seems to completely disable the d key for me; i.e. I can't close any tabs using d.

**Comment by glacambre** [comment]
Try `:bind d jsb tri.excmds.tabclose() //`.

**Comment by hejops** [comment]
That did it; thanks!

**Comment by pataquets** [comment]
I'd like to use the right-side numbers in a split keyboard,- as binds, but I've found out that numeric prefixing prevents `bind` to work.
Is there any hidden setting to toggle off numeric prefixing so  `bind 8 ...` can work?

**Comment by bovine3dom** [comment]
No, our numeric prefixes are hard coded (which I would like to change one day...). With #4439 you could remap the number keys to other keys which you could then bind.
